internal/server/server_ingest.go

```diff
@@ -108,18 +108,28 @@ func (s *Server) IngestWithTable(ctx context.Context, request *talaria.IngestWit
 		// Functions to be applied
 		funcs := []applyFunc{block.Transform(filter, s.computed...)}
 
-		// If table supports streaming, add publishing function
-		if streamer, ok := t.(storage.Streamer); ok {
-			funcs = append(funcs, stream.Publish(streamer, s.monitor))
-		}
-
 		// Partition the request for the table
 		blocks, err := block.FromRequestWithTable(request, appender.HashBy(), filter, funcs...)
 		if err != nil {
 			s.monitor.Count1(ctxTag, ingestErrorKey, "type:convert")
 			return nil, errors.Internal("unable to read the block", err)
 		}
 
+		// If table supports streaming, then stream
```

Comment:
Appending to funcs will be handled by multiApply which does the streaming. Why do we need to stream it here seperating it from other transform funcs?